iFixit/react-commerce
# Do not bundle algolia if not needed

Only bundle Algolia if the template requires it.
(Saves ~30kb gzipped in non algolia templates)

### QA

1. Visit Vercel preview
2. Verify that Algolia templates are still working correctly (navigation, browser back button etc)
